contracts/src/v0.8/vrf/dev/VRFV2PlusWrapper.sol

```diff
@@ -346,20 +374,23 @@ contract VRFV2PlusWrapper is ConfirmedOwner, TypeAndVersionInterface, VRFConsume
     int256 _weiPerUnitLink
   ) internal view returns (uint256) {
     // costWei is the base fee denominated in wei (native)
-    // costWei takes into account the L1 posting costs of the VRF fulfillment
-    // transaction, if we are on an L2.
-    uint256 costWei = (_requestGasPrice *
-      (_gas + s_wrapperGasOverhead + s_coordinatorGasOverhead) +
-      ChainSpecificUtil._getL1CalldataGasCost(s_fulfillmentTxSizeBytes));
-    // (1e18 juels/link) * ((wei/gas * (gas)) + l1wei) / (wei/link) == 1e18 juels * wei/link / (wei/link) == 1e18 juels * wei/link * link/wei == juels
-    // baseFee is the base fee denominated in juels (link)
-    uint256 baseFee = (1e18 * costWei) / uint256(_weiPerUnitLink);
-    // feeWithPremium is the fee after the percentage premium is applied
-    uint256 feeWithPremium = (baseFee * (s_wrapperPremiumPercentage + 100)) / 100;
-    // feeWithFlatFee is the fee after the flat fee is applied on top of the premium
-    uint256 feeWithFlatFee = feeWithPremium + (1e12 * uint256(s_fulfillmentFlatFeeLinkPPM));
-
-    return feeWithFlatFee;
+    // (wei/gas) * gas
+    uint256 wrapperCostWei = _requestGasPrice * s_wrapperGasOverhead;
```

Comment:
Does the wrapper need its own L1 gas cost if it's running on L2?

Reply:
my understanding is that L1 cost is per transaction. Since the coordinator calls wrapper in the same transaction, the L1 calldata cost added below for coordinator is sufficient